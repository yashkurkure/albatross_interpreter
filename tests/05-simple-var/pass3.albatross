var variable int := 0;

variable := 473;
return variable;