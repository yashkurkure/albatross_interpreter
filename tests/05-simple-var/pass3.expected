Variable declared "variable" type int
Variable written "variable" type int
Variable read "variable" type int
